lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -708,6 +708,26 @@ void IMConstPropPass::rewriteModuleBody(FModuleOp module) {
 
   auto builder = OpBuilder::atBlockBegin(body);
 
+  // Unique constants per <Const,Type> pair, inserted at entry
```

Comment:
Not for this PR, but uniqued op pool is something that is appearing several places.